src/pages/Art.jsx

```diff
@@ -4,16 +4,15 @@ import React from "react";
 class Art extends React.Component {
 	render() {
 		// Replace me with real data from props
-		const art = {
-			id: 1,
-			title: "Test Art",
-			artist: "Test Artist",
-			year: 2017,
-			image: "http://via.placeholder.com/300x300"
-		};
 
+ 	this._handleClick = () => {
+			this.props.toggleFavorite(this.props.art.id);
+ 	};
```

Comment:
You definitely don't want to define class functions inside of render, this should be placed outside of it.